# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1327776168 28800
#      Sat Jan 28 10:42:48 2012 -0800
# Node ID 311d6346a4d547e4fe0d5fa2fe04652eb3f6fa91
# Parent  727b74f512af373ff162e8c3c9ebc9b40a264eb4
unimplemented.m: Remove profile and rgbplot from list of unimplemented functions.

* unimplemented.m: Remove profile and rgbplot from list of unimplemented functions.

diff --git a/scripts/help/unimplemented.m b/scripts/help/unimplemented.m
--- a/scripts/help/unimplemented.m
+++ b/scripts/help/unimplemented.m
@@ -316,29 +316,27 @@ function list = missing_functions ()
   "plotedit",
   "plottools",
   "polyeig",
   "prefdir",
   "preferences",
   "printdlg",
   "printopt",
   "printpreview",
-  "profile",
   "profsave",
   "propedit",
   "propertyeditor",
   "publish",
   "qmr",
   "quad2d",
   "questdlg",
   "rbbox",
   "reducepatch",
   "reducevolume",
   "resample",
-  "rgbplot",
   "root",
   "rotate",
   "rotate3d",
   "selectmoveresize",
   "sendmail",
   "serial",
   "setpixelposition",
   "showplottool",
